src/main/java/edu/kis/powp/jobs2d/command/visitor/CheckShapeSelection.java

```diff
@@ -0,0 +1,27 @@
+package edu.kis.powp.jobs2d.command.visitor;
+
+public class CheckShapeSelection {
```

Comment:
Extract interface and use interface where possible. 
'CheckShapeSelection' -> 'CheckPaperShape')